hawc/apps/animal/api.py

```diff
@@ -29,7 +29,11 @@ class AnimalAssessmentViewSet(BaseAssessmentViewSet):
 
     def get_endpoint_queryset(self, request):
         published_only = get_published_only(self.assessment, request)
-        return models.Endpoint.objects.get_qs(self.assessment).published_only(published_only)
+        study_ids = [int(study_id) for study_id in self.request.query_params.get("study_ids", [])]
```

Comment:
this would fail with a 500 error if the value couldn't be cast to an int. I changed to: 

`study_ids = try_parse_list_ints(request.query_params.get("study_ids"))`